includes/Core/Util/URL.php

```diff
@@ -147,13 +149,15 @@ public static function permute_site_hosts( $hostname ) {
 			// An ASCII hostname can only be non-IDN or punycode-encoded.
 			if ( $is_ascii ) {
 				// If the hostname is in punycode encoding, add the decoded version to the list of hosts.
-				if ( 0 === strpos( $hostname, Punycode::PREFIX ) || false !== strpos( $hostname, '.' . Punycode::PREFIX ) ) {
-					$host_decoded = $punycode->decode( $hostname );
+				if ( 0 === strpos( $hostname, self::PUNYCODE_PREFIX ) || false !== strpos( $hostname, '.' . self::PUNYCODE_PREFIX ) ) {
+					// phpcs:ignore PHPCompatibility.ParameterValues.NewIDNVariantDefault.NotSet
```

Comment:
Please can you add a brief comment here explaining why we're disabling the PHPCS rule?